<?xml version="1.0"?>
<launch>

    <!-- ************************ Change DIRECTORY here ********************** -->

    <!-- Database -->
    <param name="dir_dataset" value="$(find map_db_generator)/result_fullpath" />


    <!-- Test set. Use Only Testset Mode-->
    <param name="dir_dataset_test" value="$(find map_db_generator)/result_partial" />
    <param name="image_topic_name" value="/camera2/color/image_raw/compressed" />

    <!-- ********************************************************************* -->


    <!--  PACKAGES  -->
    <param name="dir_package" value="$(find VPS_localizer)" />
    <node pkg="VPS_localizer" type="kidnapper.py" name="kidnapper" output="screen" />

</launch>